scrapy/extensions/feedexport.py

```diff
@@ -204,6 +204,7 @@ def __init__(self, settings):
             raise NotConfigured
         self.format = settings['FEED_FORMAT'].lower()
         self.export_encoding = settings['FEED_EXPORT_ENCODING']
+        self.export_compression = settings.get('FEED_EXPORT_COMPRESSION', False)
```

Comment:
I _think_ adding the default value to `default_settings.py` is enough, no need to add it here as well. And `Settings.getbool` should be used to ensure that changing the setting from environment variables, the command line and such works the same.

```suggestion
        self.export_compression = settings.getbool('FEED_EXPORT_COMPRESSION')
```